### PR TITLE
have error - Table 'cinema.movie' doesn't exist

### PR DESCRIPTION
**Pull request by Fedich09**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
-target/*
\ No newline at end of file
+target/*
+*.log
```
Comment (Ketch92) [inline .gitignore]: Github says: add an empty line man

File: src/main/java/com/dev/cinema/model/Movie.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    @Override
+    public String toString() {
+        final StringBuffer sb = new StringBuffer("Movie{");
+        sb.append("id=").append(id);
+        sb.append(", title='").append(title).append('\'');
+        sb.append(", description='").append(description).append('\'');
+        sb.append('}');
+        return sb.toString();
+    }
```
Comment (Ketch92) [inline src/main/java/com/dev/cinema/model/Movie.java:51]: можу запропонувати додати equals i hashcode

**Review by Ketch92 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (Sofasmile) [inline .gitignore]: add empty line

File: src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            session = HibernateUtil.getSessionFactory().openSession();
+            transaction = session.beginTransaction();
+            session.save(movie);
+            transaction.commit();
+            return movie;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new DataProcessingException("Can't insert Movie entity ", e);
```
Comment (Sofasmile) [inline src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java]:
```suggestion
            throw new DataProcessingException("Can't insert Movie entity " + movie, e);
```

**Review by Sofasmile — CHANGES_REQUESTED** [review]

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**